csrc/python_frontend/fusion_record.h

```diff
@@ -2078,8 +2363,8 @@ struct RandomOpRecord : RecordFunctor {
   }
 
   void operator()(FusionState& fd) final {
-    auto arg1 = fd.getFusionState(args_.at(0).index);
-    auto arg2 = fd.getFusionState(args_.at(1).index);
+    auto arg1 = fd.getFusionState(args_.at(0).index)->template as<TensorView>();
```

Comment:
Why is this being changed to casting to `TensorView`?

Reply:
Good catch! I didn't pull the latest changes before moving to the new repo.